src/tests/intellij/test_analysis.py

```diff
@@ -4,28 +4,27 @@
 
 import pytest
 
+from src.models.chrome import Chrome
+
+chrome = Chrome()
 DATA_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__)))) + "/data/"
 
 
 @pytest.mark.parametrize("app_name", json.load(open(DATA_DIR + "analysis.json")))
 @pytest.mark.parametrize("ide", ["intellij"])
 @pytest.mark.intellij
-def test_analysis_intellij(configurations, setup_intellij, app_name, analysis_data, ide):
+def test_analysis_intellij(configurations, setup_intellij, app_name, analysis_data, ide, intellij_config):
     """
     Analysis tests for intellij using various migration paths
     """
     intellij = setup_intellij
-    application_data = analysis_data[app_name]
-    expected_story_points = application_data["story_points"]
-
-    # Intellij freezes without this sleep
+    # TODO: add story point verification and report opening
     time.sleep(3)
     intellij.run_simple_analysis(app_name)
-    intellij.open_report_page(app_name)
-
-    _, html_file_location = configurations
-    if "skip_reports" not in application_data["options"]:
-        intellij.verify_story_points(
-            html_file_location=html_file_location,
-            expected_story_points=expected_story_points,
-        )
+    _, report_file_path, id = configurations
```

Comment:
There's a dedicated TC for testing opening the report, I don't think it's wise to open it for each analysis executed here, the purpose of the analysis test is to check if the analysis is passing or not, so maybe remove it and create a separate TC for it ?

Reply:
done